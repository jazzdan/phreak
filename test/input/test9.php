<?php

class C {
    static $v1 = 1;
    # old comment
    const v2 = 2;
    // new comment
    function v2 () {
        return "hello from v2! (C)";
    }
    public function __construct() {
        define(self::v2, "to jest self::v2");
    }
}
class D extends C {
    function v2 () {
        return "hello from v2! (D)";
    }
}
