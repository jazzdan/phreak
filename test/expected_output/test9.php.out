class C
{
    public static $v1 = 1
    public const v2 = 2
    public function v2()
    {
        return "hello from v2! (C)";
    }

    public function __construct()
    {
        define(self::v2, "to jest self::v2");
    }

}

class D extends C
{
    public function v2()
    {
        return "hello from v2! (D)";
    }

}



